src/pages/Payment/Payment.jsx

```diff
@@ -1,7 +1,218 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { AiOutlineCreditCard } from 'react-icons/ai';
+import { BsPiggyBank } from 'react-icons/bs';
 
-function Payment(props) {
-  return <div>Payment</div>;
+function Payment() {
+  const [isChecked, setIsChecked] = useState(false);
+
+  const checkBoxHandler = e => {
+    e.target.checked ? setIsChecked(true) : setIsChecked(false);
+  };
+
+  return (
+    <Body>
+      <PaymentWrap>
+        <PaymentTitle>결제하기</PaymentTitle>
+        <OrderInfo>
+          <SubTitle>주문 정보</SubTitle>
+          <Text>클래스 이름</Text>
+          <ClassImg />
+        </OrderInfo>
+
+        <OrderInfo>
+          <SubTitle>포인트</SubTitle>
+          <PointInput placeholder="0원" type="number" />
+          <PayBtn>전액 사용</PayBtn>
+          <TextGray>사용 가능한 포인트 50,000원</TextGray>
+        </OrderInfo>
+
+        <OrderInfo>
+          <SubTitle>결제 금액</SubTitle>
+          <TextWrap>
+            <TextGray>총 상품 금액</TextGray>
+            <TextGray>45,000원</TextGray>
+          </TextWrap>
+
+          <TextWrap>
+            <TextGray>포인트 적용 금액</TextGray>
+            <TextGray>- 45,000원</TextGray>
+          </TextWrap>
+
+          <Line />
+
+          <TextWrap>
+            <Text>최종 가격</Text>
+            <TextBigBold>0원</TextBigBold>
+          </TextWrap>
+        </OrderInfo>
+
+        <OrderInfo>
+          <SubTitle>결제 방식</SubTitle>
+          <PayWrap>
+            <PayMethod>
+              <AiOutlineCreditCard
+                style={{ position: 'relative', top: '2px', marginRight: '3px' }}
+              />
+              카드 결제
+            </PayMethod>
+            <PayMethod>
+              <BsPiggyBank
+                style={{ position: 'relative', top: '2px', marginRight: '3px' }}
+              />
+              무통장 입금
+            </PayMethod>
+            <PayMethod>카카오페이</PayMethod>
+            <PayMethod>네이버페이</PayMethod>
+          </PayWrap>
+          <AgreeLabel onChange={checkBoxHandler}>
```

Comment:
```suggestion
          <AgreeLabel onChange={e => setIsChecked(e.target.checked)}>
```
결국 e.target.checked 값이 boolean 값이기 때문에
위와 같은 방법으로 작성해 주는 것이 더 좋겠습니다!